src/plugins/data/public/query/query_string/language_service/lib/query_result.tsx

```diff
@@ -143,7 +143,14 @@ export function QueryResult(props: { queryStatus: QueryStatus }) {
       data-test-subj="queryResultError"
     >
       <EuiPopoverTitle>ERRORS</EuiPopoverTitle>
-      <div style={{ width: '250px' }} className="eui-textBreakWord">
+      <div
+        style={{
+          width: '250px',
+          maxHeight: '250px',
+          overflowY: 'auto',
+        }}
```

Comment:
nit: not blocking, but unclear to me why this is inline

Reply:
Fixed

Reply:
Sorry, I meant inline styles vs css, but given that this was pre-existing, not a blocker